modules/lang-painless/build.gradle

```diff
@@ -77,13 +77,15 @@ dependencies {
   docImplementation project(':server')
   docImplementation project(':modules:lang-painless')
   docImplementation 'com.github.javaparser:javaparser-core:3.18.0'
+  docImplementation 'org.jsoup:jsoup:1.13.1'
```

Comment:
What's this do?

Reply:
It's used to [clean html tags](https://github.com/jhy/jsoup/).